src/runtime/composables/content.ts

```diff
@@ -71,12 +71,12 @@ export const useContent = () => {
   /**
    * Next page from `surround`.
    */
-  const next = computed(() => surround.value?.[1])
+  const next = computed(() => surround.value[1])
 
   /**
    * Previous page from `surround`.
    */
-  const prev = computed(() => surround.value?.[0])
+  const prev = computed(() => surround.value[0])
```

Comment:
In which case? As I can experiment, there is always a value and the object could be `[null, null]`

Reply:
`surrounds` is set inside `documentDriven.ts` only if HTTP call succeeds, if HTTP call fails surround will be `null`

Also, it will be null if users disable `surrounds` in document driven.

Reply:
Ho yes, that's a good point.